src/main/java/org/fundacionjala/coding/cynthia/HighestAndLowest.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.cynthia;
+
+/**
+ * Created by CynthiaTerrazas on 6/12/2017.
+ */
+public final class HighestAndLowest {
+
+    /**
+     * constructor.
+     */
+    private HighestAndLowest() {
+    }
+
+    /**
+     * @param cadena number.
+     * @return calculate value.
+     */
+    public static String calculated(String cadena) {
+        String[] setNumbers = cadena.split(" ");
+        int mayor = -100;
+        int menor = 100;
```

Comment:
Instead of using these numbers
please use these constants
```java
        int mayor = Integer.MIN_VALUE;
        int menor = Integer.MAX_VALUE;
```